keymaster.spec

```diff
@@ -1,5 +1,5 @@
 Name:           keymaster
-Version:        1.15.1
+Version:        1.15.2
```

Comment:
Should the `Makefile` version be bumped as well?

Reply:
Yes, thanks for the reminder. I will update and then land.